src/auth_service.js

```diff
@@ -0,0 +1,85 @@
+// Store token and expiry in localStorage
+const TOKEN_STORAGE_KEY = 'auth_token';
+const EXPIRY_STORAGE_KEY = 'token_expiry';
+
+// Your API details
+const AUTH_URL = process.env.REACT_APP_AUTH_URL;
+const header = process.env.REACT_APP_AUTH_CREDENTIALS;
+const BASIC_AUTH_CREDENTIALS = btoa(header); // Base64 encoding for Basic Auth
```

Comment:
instead of having repeated variables (username, password, and authCredentials which is just username:password) - can we simplify the env vars and construct the header from the provided user/pass env vars?